web/gds-admin-ui/src/pages/app/dashboard/Status.js

```diff
@@ -2,18 +2,22 @@ import React from 'react';
 import { Doughnut } from 'react-chartjs-2';
 import { Card, Row, Col } from 'react-bootstrap';
 import { capitalizeFirstLetter, convertCountsToPercentages } from '../../../utils';
+import { Status as STATUS } from '../../../constants';
 
 const Status = ({ statuses }) => {
     const colors = ['#0acf97', '#727cf5', '#fa5c7c'];
 
-    const getDonutChartData = () => {
+    const statusesPercentages = () => {
         if (statuses && typeof statuses === "object") {
-            return Object.values(convertCountsToPercentages(statuses))
+            return convertCountsToPercentages(statuses)
         }
 
-        return []
+        return {}
     }
 
+    const getDonutChartData = () => Object.values(statusesPercentages())
+
```

Comment:
@elysee15 what I recommend is that we do the following:

```javascript
const getDonutChartData = () => Object.values(statusesPercentages())
const statusPercents = () => Object.fromEntries(statusPercentages().map((key, val) => [key, val*100.0]))
```

Then use `statusPercents` in the `<span>`, e.g. `<span>{statusPercents()[STATUS.PENDING_REVIEW] + '%'}</span>